Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,212 @@
+
+let word = allTheWords.randomElement() ?? "bob"
+
+// creating animation function:
```

Comment:
Make sure to clean up your code before you submit, that includes removing all notes that you've left for yourself or any line of code that isn't being used :)